extensions/Worker.Extensions.CosmosDB/src/CosmosDBTriggerAttribute.cs

```diff
@@ -6,6 +6,7 @@
 
 namespace Microsoft.Azure.Functions.Worker
 {
+    [BindingCapabilities("FunctionLevelRetry")]
```

Comment:
Can we introduce a public constants class to contain these capabilities?

``` CSHarp
public static class BindingCapabilities
{
    public const string FunctionLevelRetry = nameof(FunctionLevelRetry);
}
```

Although if this interferes with use `[BindingCapabilities()]` attribute due to name, then name the const class something else.

Reply:
Would be good to have this as part of the changes, for the name, I actually like `KnownBindingCapabilities`, thoughts?